### PR TITLE
SMA-220/SMA-223: Paving the way for the addition of the dark mode

### PR DESCRIPTION
**Pull request by qnga**
**What's this do?**
- Remove all (legacy) code from `simplified-ui-theme`
- Reorganize styles and themes, enable dark mode
- Enable forced dark mode in webviews

Note: this is a preliminary work, the dark mode is not supposed to be correct.

**Why are we doing this? (w/ JIRA link if applicable)**
https://jira.nypl.org/browse/SMA-220
https://jira.nypl.org/browse/SMA-223

**How should this be tested? / Do these changes have associated tests?**
- Check nothing is broken.
- Check the app is able to switch to dark mode.
- Check webviews (excepting those from R2) are dark when your device is in dark mode.

**Dependencies for merging? Releasing to production?**
None

**Have you updated the changelog?**
No, I'll do that once there is actually something new working.

**Has the application documentation been updated for these changes?**
N/A

**Did someone actually run this code to verify it works?**
Yes, I did, excepted for SAML webviews. I believe it works and we'll be able to check once the whole work is done.

### CODE REVIEW

File: simplified-cardcreator/src/main/java/org/nypl/simplified/cardcreator/ui/EULAFragment.kt [hunk 1]
```diff
@@ -19,13 +20,14 @@ class EULAFragment : Fragment() {
...
     savedInstanceState: Bundle?
-  ): View? {
+  ): View {
     _binding = FragmentEulaBinding.inflate(inflater, container, false)
     return binding.root
   }
 
   override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
     super.onViewCreated(view, savedInstanceState)
+    WebViewUtilities.setForcedDark(binding.webview.settings, resources.configuration)
```
Comment (MalcolmMcFly) [inline simplified-cardcreator/src/main/java/org/nypl/simplified/cardcreator/ui/EULAFragment.kt:30]: Good!

**Comment by MalcolmMcFly** [comment]
@qnga This code looks very good. To test, we just set our device to dark mode and run the app to ensure it shows dark mode to match the system setting?

**Comment by MalcolmMcFly** [comment]
This looks good @qnga. As you stated above, I am aware that is not completed. I saw these: 
Screenshot_20210907-214625
Screenshot_20210907-214635

Approved 👍

**Review by MalcolmMcFly — APPROVED** [review]

**Comment by qnga** [comment]
Thanks for reporting issues!

**Review by io7m — APPROVED** [review]
Glad to see a lot of the old theme junk going away.

**Pull request merged**